configs/a64fx.yaml

```diff
@@ -245,6 +247,123 @@ Latencies:
     - STORE_ADDRESS_SVE
     Execution-Latency: 6
     Execution-Throughput: 1
+  11:
+    Instruction-Groups:
```

Comment:
Can a comment be added explaining what all these opcodes are (i.e. NEON and SVE indexed FMLA) and why they are being handled in this way as opposed to the normal way

Reply:
+1

Reply:
Added a brief comment

Reply:
Sorry to add to this. Could a `/** NOTE: */` or similar be added too saying that any changes to Capstone will likely break this functionality (as Opcode values will change)

Reply:
I think the Capstone enum "names" can be used instead of the raw numbers  which <em>may</em> prevent breakages if Capstone changes. I think I successfully tried this a while ago but I could be wrong.

Reply:
https://uob-hpc.github.io/SimEng/developer/arch/supported/riscv.html#instruction-groups

See bottom paragraph of this section

Reply:
Given this is in the YAML file and not in the code itself, we can't use the enum name as (I'm fairly sure) you can't convert a string to an enum name quickly. Resolving as initial issues have been amended